pages/ecosystem.vue

```diff
@@ -248,6 +222,13 @@ const filteredMembers = computed<Member[]>(() => {
 
   let filteredMembers = members;
 
+  // Tier filter
+  if (tierFilters.value.length > 0) {
```

Comment:
they're almost similar... 

the tier filter checks `=> member.tier.includes(filter))` (the `.tier` property)
the category filter check `=> member.labels.includes(filter))` (the `.labels` property)

Reply:
Yeah, I know, I was thinking on doing [bracket notation](https://stackoverflow.com/questions/4244896/accessing-an-object-property-with-a-dynamically-computed-name#:~:text=notation%3A%20something.bar-,Bracket%20notation,-%3A%20something%5B%27bar), but it's true that [it can be dangerous on some cases](https://github.com/eslint-community/eslint-plugin-security/blob/main/docs/the-dangers-of-square-bracket-notation.md), so I'm ok leaving it this way for now